src/main/java/Deadlines.java

```diff
@@ -0,0 +1,42 @@
+
+public class Deadlines extends Task {
+    private final String dateTime;
+
+    private Deadlines(int taskId, String taskName, 
+            String dateTime, boolean isDone) {
+
+        super(taskId, taskName, isDone);
+        this.dateTime = dateTime;
+    }
+
+    public Deadlines(int taskId, String taskName, String dateTime) {
+        super(taskId, taskName);
+        this.dateTime = dateTime;
+    }
+
+    public String getDateTime() {
+        return this.dateTime;
+    }
+
+    @Override
+    public String taskWithSymbol() {
+        return "[D]" 
+                + ((this.isDone) ? "[✓]" : "[✗]")

```

Comment:
Can declare a final variable TICK to store its symbol ✓, so it can be referenced more easily (without copying the symbol again)

Reply:
This too as well.